extensions/interactions/DragAndDropSortInput/directives/oppia-short-response-drag-and-drop-sort-input.directive.ts

```diff
@@ -38,7 +40,12 @@ angular.module('oppia').directive('oppiaShortResponseDragAndDropSortInput', [
           return true;
         };
         ctrl.$onInit = function() {
-          ctrl.answer = HtmlEscaperService.escapedJsonToObj($attrs.answer);
+          const answer = HtmlEscaperService.escapedJsonToObj($attrs.answer);
+          const interactionchoices = HtmlEscaperService.escapedJsonToObj(
```

Comment:
This will be choices only with html tags in it

![Screenshot from 2021-05-26 01-12-01](https://user-images.githubusercontent.com/62715691/119558938-a9329600-bdbf-11eb-8c16-81a8ae153982.png)

And this is interactionchoices with subtitled value i.e.(html+contentId)

![Screenshot from 2021-05-26 01-12-23](https://user-images.githubusercontent.com/62715691/119559050-c9faeb80-bdbf-11eb-901a-48843f0ad7d6.png)


We will need html + contentId value to show the answers as we are mapping the answers with contentId. So we will need interactionchoices which has both.